bayesian/utility/v2/component_analyses.py

```diff
@@ -240,6 +241,28 @@ def get_known_unknown_pkgs(
     return stack_recommendation, unknown_pkgs
 
 
+def get_known_clair_pkgs(
```

Comment:
This entire function needs to be re written. This is trying to parse a wrong data/information.